docs/user_guides/fs/storage_connector/creation/snowflake.md

```diff
@@ -16,18 +16,39 @@ When you're finished, you'll be able to query the database using Spark through H
 
 Before you begin this guide you'll need to retrieve the following information from your Snowflake account and database, the following options are **mandatory**:
 
-- **Snowflake Connection URL:** Consult the documentation of your target snowflake account to determine the correct connection URL. This is usually some form of your [Snowflake account identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html). For example:
+- **Snowflake Connection URL:** Consult the documentation of your target snowflake account to determine the correct 
+  connection URL. There are multiple formats supported by Snowflake. This is usually some form of your [Snowflake account 
+  identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html) or an 
+[account URL](https://docs.snowflake.com/en/user-guide/organizations-connect.html#connecting-with-a-url).
```

Comment:
Same comment as for the frontend PR "account url" is not a thing - if you look at the Snowflake documentation, the URL looks like this:https://docs.snowflake.com/en/user-guide/admin-account-identifier.html

The first part can be a random generated string or it can be something specific to the organization. But the format is the same.

Reply:
To me it seems like they are using account URL terminology along with account identifiers as per the formats mentioned here https://docs.snowflake.com/en/user-guide/organizations-connect.html#standard-account-urls . 
Thats why I thought it was needed to explicitly mention account identifier and account URL. Account URLs again are having different url formats.
Secondly from the Snowflake UI, it allows to just copy the account URL, which I think is easier for snowflake users to do.(example gif)

![snaccounturl](https://user-images.githubusercontent.com/55157590/192482501-300318e0-f8c0-4747-aead-89055a67467e.gif)

But if you think it is better to stick with just account identifiers I can remove it.

Reply:
removed account url naming